plugin.video.fosdem/resources/lib/addon.py

```diff
@@ -30,8 +28,15 @@ def years():
     return range(year, START_YEAR - 1, -1)
 
 
+def get_setting_int(name):
+    val = getSetting(plugin.handle, name)
+    if not val:
+        val = '0'
+    return int(val)
+
+
 def get_format():
-    return FORMATS[addon.getSettingInt('format')]
+    return FORMATS[get_setting_int('format')]
```

Comment:
The is an `Addon.getSettingInt()`. See https://romanvm.github.io/Kodistubs/_autosummary/xbmcaddon.html